example.py

```diff
@@ -216,6 +225,9 @@ def get_arguments() -> Tuple[argparse.ArgumentParser, argparse.Namespace]:
     parser.add_argument(
         "--gen2", "-g2", action="store_true", help="Force Gen 2 (RPC) device"
     )
+    parser.add_argument(
+        "--debugger", "-deb", action="store_true", help="Enable debug level for logging"
```

Comment:
This not really enable a debugger, just set a debug log, so it would be better to call it `--debug`